### PR TITLE
Define Hookups Feature

### PR DESCRIPTION
**Pull request by zspencer**
As we expand to support more external services, we're going to want a way for external services to be configured on a Space-by-Space basis.

### CODE REVIEW

File: features/hookups.feature [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    When the Space Owner removes that Hookup
+    Then the Hookup can not be used by Furniture in the Space
+
+  # A Hookup that is being used by Furniture probably needs a bit more thought
+  # put into it for removal. So we're going to... not let people do that that
+  # for Andromeda. Unless someone is eager to take it!
+  #
+  # See "Scenario: Removing an in-use Hookup" for future plans.
+  @unstarted @andromeda
+  Scenario: Cnanot remove an in-use Hookup
```
Comment (user512) [inline features/hookups.feature:44]:
```suggestion
  Scenario: Cannot remove an in-use Hookup
```

**Review by user512 — APPROVED** [review]

**Comment by user512** [comment]
Might need to fix the reference for the moved file.

File: features/hookups.feature (hunk 1 above)
Comment (zspencer) [reply features/hookups.feature:44]: Thank you! nice catch!

**Comment by zspencer** [comment]
Fixed! Thank you!

**Review by user512 — APPROVED** [review]

**Pull request merged**